openfisca_survey_manager/tests/test_add_survey_to_collection.py

```diff
@@ -9,7 +9,7 @@
 from openfisca_survey_manager.scripts.build_collection import add_survey_to_collection
 from openfisca_survey_manager.input_dataframe_generator import set_table_in_survey
 
-# Travis, Gitlab runner, Gihub Action and circle has env variable "CI" set by default
+# GitLab Runner and GitHub Actions have env variable "CI" set by default
 if 'CI' in os.environ:
```

Comment:
En une ligne :
```suggestion
is_in_ci = 'CI' in os.environ
```

Reply:
Done in 430b7875394cadbf842526e2798be43b6f839e5a (to update the `else` line under the suggestion).